packages/xo-web/src/common/xo/migrate-vms-modal/index.js

```diff
@@ -262,6 +262,11 @@ export default class MigrateVmsModalBody extends BaseComponent {
   _toggleSmartVifMapping = () =>
     this.setState({ smartVifMapping: !this.state.smartVifMapping })
 
+  compareContainers = (pool1, pool2) => {
+    const poolIds = map(this.props.vms, (value, _) => value.$poolId)
```

Comment:
```js
map(this.props.vms, '$pool')
```

[Use `$pool` instead of `$poolId`](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/xapi-object-to-xo.js#L807)